### ISSUE TITLE
Disable preview for files under moratorium

### ISSUE THREAD

**Issue by ukarst** [ICOS-Carbon-Portal/data#305]
Datasets (csv files) under moratorium should not have a preview, e.g. https://meta.icos-cp.eu/objects/iG7Zxt5xxyuNt0u7a6EMhWHz
Via preview the data can then even be downloaded despite the moratorium.

**Comment by mirzov** [comment]
Good catch! For now, have deleted the corresponding .cpb files from the server, to prevent data access, and will soon hide the preview tab on the landing pages of objects under moratorium, but more work is needed in the portal app and data back end.
